cpp/inc/bond/protocol/compact_binary.h

```diff
@@ -61,17 +61,21 @@
                                             '---'---'---'---'---'---'---'---'
                                                                           0
 
+                                            falsy booleans fields are not encoded
```

Comment:
@chwarr I have an e-mail notification about a comment on this statement, but it doesn't appear on GitHub anymore. Was the comment deleted?

Reply:
I moved it into a review. Should be visible now, under that review.